docs/application/native/guides/text-input/autofill.md

```diff
@@ -295,24 +275,24 @@ To retrieve the data previously entered by the user and stored in an Autofill se
 3. To get the data previously entered by the user and stored in an Autofill service, request using the `autofill_fill_request()` function:
 
     ```c
-    /* Send request to get autofill fill response */
+    /* Send request to get fill response */
     int ret = autofill_fill_request(ah, vi_h);
     if (ret != AUTOFILL_ERROR_NONE)
         /* Error handling */
     ```
 
-4. Incase, the Autofill view handle is not needed in your application, you must destroy the autofill view handle using
+4. Incase, the Autofill view handle is not needed in your application, you must destroy the Autofill view handle using
```

Comment:
@mijin85cho, 
It looks like it is common step for "Autofill Data and Autofill Authentication Information" and "Data Entered and Stored in Autofill Service". Can't these move to "Release Resources" section?

Comment is not fixed.

Reply:
Moved into "Release Resource" section.